dalm/pipelines/reading_comprehension_pipeline.py

```diff
@@ -0,0 +1,383 @@
+    parser.add_argument("--csv_column", type=str, help="Column to read from the CSV file")
+    parser.add_argument("--no_chunk", action="store_true", help="whether to NOT chunk the input files or not")
+    parser.add_argument("--num_train_epochs", type=int, default=1, help="number of epochs to train the generator")
+    parser.add_argument("--split", type=str, default="train", help="split to be used for training")
+    parser.add_argument("--size_valid_set", type=int, default=1000, help="size of the validation set (STREAMING ONLY)")
+    parser.add_argument("--validation_split", type=float, default=0.05, help="validation split")
+    parser.add_argument("--streaming", action="store_true", help="whether to use streaming or not")
+    parser.add_argument("--shuffle_buffer", type=int, default=10000, help="shuffle buffer size (STREAMING ONLY)")
+    parser.add_argument("--seq_length", type=int, default=2600, help="sequence length to be used for training")
+    parser.add_argument(
+        "--num_workers", type=int, default=4, help="number of workers to be used for data loading during training"
+    )
+    parser.add_argument("--eval_steps", type=int, default=200, help="number of steps to evaluate the model")
+    parser.add_argument("--logging_steps", type=int, default=1000, help="number of steps to log the model")
+    parser.add_argument("--per_device_train_batch_size", type=int, default=1, help="batch size to be used for training")
+    parser.add_argument(
+        "--per_device_eval_batch_size", type=int, default=1, help="batch size to be used for evaluation"
+    )
+    parser.add_argument("--gradient_accumulation_steps", type=int, default=1, help="gradient accumulation steps")
+    parser.add_argument(
+        "--no_gradient_checkpointing", action="store_true", help="whether to disable gradient checkpointing"
+    )
+    parser.add_argument("--group_by_length", action="store_true", help="whether to group the dataset by length or not")
+    parser.add_argument("--no_packing", action="store_true", help="whether to disable packing or not")
+    parser.add_argument("--lora_alpha", type=int, default=512, help="lora alpha")
+    parser.add_argument("--lora_dropout", type=float, default=0.05, help="lora dropout")
+    parser.add_argument("--lora_r", type=int, default=256, help="lora r")
+    parser.add_argument("--learning_rate", type=float, default=1e-4)
+    parser.add_argument("--lr_scheduler_type", type=str, default="cosine")
+    parser.add_argument("--num_warmup_steps", type=int, default=100)
+    parser.add_argument("--weight_decay", type=float, default=0.05)
+    parser.add_argument("--optimizer_type", type=str, default="paged_adamw_32bit")
+    parser.add_argument("--model_output_dir", type=str, default="model_output_dir")
+    parser.add_argument("--neftune_noise_alpha", type=int, default=5)
+    parser.add_argument(
+        "--log_with",
+        type=str,
+        help="tracker backend to be used",
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "--log_with",
        type=str,
        default="none",
        help="tracker backend to be used",
    )
```

See comments here: https://github.com/arcee-ai/DALM/pull/74#issuecomment-1834297869